RecoHI/HiJetAlgos/test/makeDBfromTxt.py

```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+ivars = VarParsing.VarParsing('standard')
+
+ivars.register ('outputTag',
+                mult=ivars.multiplicity.singleton,
+                mytype=ivars.varType.string,
+                info="for testing")
+ivars.outputTag="demo"
+
+ivars.register ('inputFile',
+                mult=ivars.multiplicity.singleton,
+                mytype=ivars.varType.string,
+                info="for testing")
+
+ivars.register ('outputFile',
+                mult=ivars.multiplicity.singleton,
+                mytype=ivars.varType.string,
+                info="for testing")
+
+ivars.inputFile="/afs/cern.ch/user/y/ylai/ue_calibrations_pf_data.txt"
```

Comment:
private user files should not be used as input.
just a plain file name without the path to the user area is OK